src/pages/questionPage.js

```diff
@@ -21,10 +22,48 @@ export const initQuestionPage = () => {
 
   const answersListElement = document.getElementById(ANSWERS_LIST_ID);
 
+  const correctAnswer = currentQuestion.correct
+  // const correctAnswer = quizData.questions[quizData.currentQuestionIndex.correct]
+
   for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
     const answerElement = createAnswerElement(key, answerText);
     answersListElement.appendChild(answerElement);
+    
+    answerElement.addEventListener('mouseenter', function(event) {
+      event.target.style.transform = "scale(1.1)"
+      event.target.style.color = 'orange'
+      setTimeout(function() {
+        event.target.style.color = "";
+        event.target.style.transform = "";
+      }, 500);
+    }
+    , false);
+    answerElement.addEventListener('click', function() {
+      // for(let i = 0; i < answersListElement.length; i++) {
+      if(answerElement === correctAnswer){
```

Comment:
Logic about which answer is correct belongs in the 'views' directory, therefore ideally it should look something like this:
`const response = onAnswerClick();
 answerElement.appendChild(response)`

Where `onAnswerClick` is a function in views/questionView.js that returns an element containing what you want to show to the user.